src/IridiumSBD.cpp

```diff
@@ -171,6 +171,24 @@ void IridiumSBD::adjustATTimeout(int seconds)
    this->atTimeout = seconds;
 }
 
+// Tweak SBD Session Timeout
+int IridiumSBD::adjustSBDSessionTimeout(int seconds)
+{
+   if ((seconds >= this->atTimeout) || seconds == 0)
+      diagprint(F("SBD commands that do not complete before AT timeout will return ISBD_PROTOCOL_ERROR\r\n"));
+   this->sbdSessionTimeout = seconds;
```

Comment:
Is it dangerous to update `this->sbdSessionTimeout` here - before the `send` and `waitForATResponse` has been attempted?
Would it be better to update it only if the `send` is successful? Perhaps move it to after the `if`?

```
   this->sbdSessionTimeout = seconds;
   return ISBD_SUCCESS;
```